python/triton/experimental/gluon/language/amd/cdna3/__init__.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING
+
+from triton.experimental.gluon.language import _core as ttgl
+from triton.experimental.gluon.language._core import builtin
+from triton._C.libtriton import ir
+
+if TYPE_CHECKING:
+    from ..._semantic import GluonSemantic
+
+__all__ = ["buffer_load", "buffer_store"]
+
+
+@builtin
+def buffer_load(ptr, offsets, mask=None, other=None, stride=None, cache=None, _semantic=None):
```

Comment:
Let's do not expose `stride` for now. So please drop it from the parameters for now.